Tools/bldsys/include/bs_dir_names.cmake

```diff
@@ -41,12 +41,12 @@ endif(NOT DEFINED GFX_DEVELOPMENT_DIR)
 #
 # These will be used later to help with separation of components.
 #
-bs_set_if_undefined(BS_DIR_SOURCE          "${GFX_DEVELOPMENT_DIR}/Source")
+bs_set_if_undefined(BS_DIR_SOURCE          "${CMAKE_SOURCE_DIR}/..")
 
-bs_set_if_undefined(BS_DIR_COMMON          "${BS_DIR_SOURCE}/Common")
-bs_set_if_undefined(BS_DIR_GMMLIB          "${BS_DIR_SOURCE}/GmmLib")
-bs_set_if_undefined(BS_DIR_INC             "${BS_DIR_SOURCE}/inc")
+bs_set_if_undefined(BS_DIR_COMMON          "${BS_DIR_SOURCE}/gmmlib/Source/Common")
```

Comment:
I hope that everyone understands that we need to have pkg-config identification for _installed_ GmmLib? Whether it is dynamic or static it actually does not matter.

Reply:
This is still a temp solution before we can change to use dynamic link GmmLib.

Reply:
Yep, this is temporary. This should be removed when we've switched over to pkg-config.

Reply:
ok, I just wanted to double check target goal:).